crates/matrix-sdk/src/http_client/mod.rs

```diff
@@ -231,6 +231,15 @@ pub struct TransmissionProgress {
     pub total: usize,
 }
 
+/// Progress of an operation in abstract units.
```

Comment:
Can you explain in the doc comment why the units are abstract / what does it mean?